.update-locales.sh

```
@@ -0,0 +1,37 @@
+#!/bin/sh
+COMPONENTS=eventyay/eventyay-plugin-eventyay-business
+DIR=eventyay_business/locale
+# Renerates .po files used for translating the plugin
+set -e
+set -x
+
+# Lock Weblate
+for c in $COMPONENTS; do
+	wlc lock $c;
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)